plugins/woocommerce/src/Internal/Features/ProductBlockEditor/ProductTemplates/ProductVariationTemplate.php

```diff
@@ -137,7 +137,7 @@ private function add_general_group_blocks() {
 				'order'      => 20,
 				'attributes' => array(
 					'property' => 'description',
-					'label'    => __( 'Note <optional />', 'woocommerce' ),
+					'label'    => __( 'Note (optional)', 'woocommerce' ),
```

Comment:
I'm afraid the fix isn't super straightforward. The `<optional />` string actually acts like a placeholder to render a component [here](https://github.com/woocommerce/woocommerce/blob/92837fc7060eac562181858d2be1fff1cbe0c60d/packages/js/product-editor/src/blocks/product-fields/summary/edit.tsx#L91-L107).

Reply:
Does this PR require any actions? Original issue has already been closed ((optional) has been removed from the label text).